spec/services/integration_tests/test_runner_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe IntegrationTests::TestRunner, type: :request do
+  let(:spreadsheet_file) { Rails.root.join('spec/fixtures/integration_test_data.xlsx') }
+  let(:spreadsheet) { Roo::Spreadsheet.open(spreadsheet_file.to_s) }
+  let(:worksheet_names) { spreadsheet.sheets.select { |name| name.starts_with?('Test - ') } }
+  let(:headers) { { 'CONTENT_TYPE' => 'application/json' } }
+
+  describe '#steps' do
+    def create_assessment(payload)
+      post assessments_path, params: payload[:assessment].to_json, headers: headers
+      JSON.parse(response.body)['objects'].first['id']
+    end
+
+    def run_spreadsheet(spreadsheet_name)
+      payload = IntegrationTests::WorksheetParser.call(spreadsheet.sheet(spreadsheet_name))
+      assessment_id = create_assessment(payload)
+      described_class.steps(assessment_id, payload).each do |step|
+        post step.url, params: step.params.to_json, headers: headers
```

Comment:
Is it worth just putting a test in here that the response was successful? Something like:
```ruby
    it 'returns success' do
      expect(response).to be_successful
    end
```